### PR TITLE
PMM-13141 Added feature compatibility version collector

### PR DESCRIPTION
**Pull request by cfsalguero**
[PMM-13141](https://jira.percona.com/browse/PMM-13141 

Added a new collector for Feature compatibility version,

[PMM-13141]: https://perconadev.atlassian.net/browse/PMM-13141?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: exporter/exporter.go [hunk 1]
```diff
@@ -70,6 +70,9 @@ type Opts struct {
 	EnableCollStats          bool
 	EnableProfile            bool
 	EnableShards             bool
+	EnableFCV                bool // Feature Compatibility Version.
+
+	FCVInterval time.Duration
```
Comment (BupycHuk) [inline exporter/exporter.go:75]: I'm not sure if we need separate interval for FCV.

File: exporter/feature_compatibility_version_collector.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+		if err := res.Decode(&m); err != nil {
+			ch <- prometheus.NewInvalidMetric(prometheus.NewInvalidDesc(err), err)
+			return
+		}
+
+		d.lastScrape = d.now()
+
+		rawValue := walkTo(m, []string{"featureCompatibilityVersion", "version"})
+		if rawValue != nil {
+			version, err := strconv.ParseFloat(fmt.Sprintf("%v", rawValue), 64)
```
Comment (BupycHuk) [inline exporter/feature_compatibility_version_collector.go:90]: wouldn't it better to use hashicorps version library to parse it?

**Review by BupycHuk — COMMENTED** [review]
LGTM, just few comments

File: exporter/exporter.go (hunk 1 above)
Comment (igroene) [reply exporter/exporter.go:75]: this value changes only after upgrades so it should have a big interval to avoid unnecessary reads

Comment (BupycHuk) [reply exporter/exporter.go:75]: that's true, but it's not a good practice in exporter development. Is this operation generate a big load on system?

Comment (igroene) [reply exporter/exporter.go:75]: no, not really. We can use one of the default collection intervals

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**